google/resource_sql_database_instance_test.go

```diff
@@ -9,6 +9,7 @@ import (
 	"github.com/hashicorp/terraform/helper/acctest"
 	"github.com/hashicorp/terraform/helper/resource"
 	"github.com/hashicorp/terraform/terraform"
+
```

Comment:
weird that this got added

Reply:
I think this is a concurrency thing, that `goimports` behaviour changes based on the ordering of generated files.